io/src/main/java/org/commonjava/maven/ext/io/XMLIO.java

```diff
@@ -56,7 +56,7 @@ public XMLIO()
 
             transformer = TransformerFactory.newInstance().newTransformer();
             transformer.setOutputProperty( OutputKeys.INDENT, "yes");
-            transformer.setOutputProperty( OutputKeys.ENCODING, StandardCharsets.UTF_8.toString());
+            transformer.setOutputProperty( OutputKeys.ENCODING, StandardCharsets.UTF_8.name() );
```

Comment:
Why this change?

Reply:
Because `name()` (Returns this charset's canonical name) should be what we want, not `toString()` (Returns a string describing this charset). If `name()` and `toString()` match that's more of a coincidence:

Reply:
I see what you are saying but 
```
    public final String toString() {
        return name();
    }
```
hence the change isn't really required?

Reply:
I think it is just a coincidence that the code works. The argument that is supposed to be passed is the charset name and nothing would prevent `toString()` from returning some other information. The API says:

```every charset has a canonical name and may also have one or more aliases. The canonical name is returned by the name method of this class. Canonical names are, by convention, usually in upper case.```

Reply:
Ok fine

Reply:
Ok.